src/Microsoft.AspNet.Mvc.Core/TaskCache.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.Mvc
+{
+    public static class TaskCache
+    {
+#if DNX451 || DNX452
```

Comment:
Something is wrong here: :https://github.com/aspnet/Mvc/blob/9915ea38097857cc8046b9cceaec7ad6bd0b9b11/test/Microsoft.AspNet.Mvc.FunctionalTests/ViewEngineTests.cs 

for this test:

```RazorViewEngine_UsesExpandersForLayouts```